src/test/java/tracko/logic/parser/FindItemCommandParserTest.java

```diff
@@ -26,10 +24,10 @@ public void parse_validArgs_returnsFindItemCommand() {
         // no leading and trailing whitespaces
         FindItemCommand expectedFindItemCommand =
                 new FindItemCommand(new ItemContainsKeywordsPredicate(Arrays.asList("Chair", "Keychain")));
-        assertParseSuccess(parser, "Chair Keychain", expectedFindItemCommand);
+        CommandParserTestUtil.assertParseSuccess(parser, "Chair Keychain", expectedFindItemCommand);
```

Comment:
Thank you might have forgotten to remove the CommandParserTestUtil here. Use the method directly. Do check for the other instances